### PR TITLE
Adds dynamite to the game

### PR DESCRIPTION
**Pull request by bryantfhayes**
Adds dynamite which causes EVENT_BOOM

### CODE REVIEW

File: cogvents.py [hunk 1]
```diff
@@ -25,15 +25,23 @@ def AddListener(event, fn):
     events[event].append(fn)
 
 def Emit(e, data=None):
-    for event in events[e]:
-        event(data)
+    if e in events:
```
Comment (CALEBPENTECOST) [inline cogvents.py]: Have you looked at line 13?

File: cogvents.py [hunk 2]
```diff
@@ -25,15 +25,23 @@ def AddListener(event, fn):
...
 #                 Add Functions                    #
 ####################################################
 def fnStart(data):
     print("Start!")
 
+def fnDynamite(data):
+    import random
+    odds = (random.randint(1,100))
+    if odds <= 20:
+        print("Dynamite blew!")
```
Comment (CALEBPENTECOST) [inline cogvents.py]: I propose a change of this string to: "And boom goes the dynamite"

File: cogvents.py (hunk 1 above)
Comment (bryantfhayes) [inline cogvents.py]: haha like i SAID IN THE COMMENT, it was a gameplay bug fix

Comment (CALEBPENTECOST) [inline cogvents.py]: Acceptable.

File: cogvents.py (hunk 2 above)
Comment (CALEBPENTECOST) [inline cogvents.py]: Also, can you add newlines to each print?

File: cogvents.py [hunk 3]
```diff
@@ -25,15 +25,23 @@ def AddListener(event, fn):
...
+        Emit("Event_Boom")
```
Comment (CALEBPENTECOST) [inline cogvents.py]: This emit uses different quotes from the original one.

Comment (bryantfhayes) [inline cogvents.py]: That's my secret strategy to win. replace all " with Event_Win

**Comment by Lapitz** [comment]
We can change it if you want but we were thinking of not allowing any change requests so that the game doesn't take 5 years to finish. If you don't like the code, reject it.

**Pull request merged**